src/actions.js

```diff
@@ -139,7 +140,18 @@ function optSetIncDecStep(label = 'Value', def, min, max, step = 1) {
 			step: step,
 			required: true,
 			range: true,
-			isVisible: (options) => options.op === 's',
+			isVisible: (options) => options.op === 's' && !options.useVar,
+		},
+		{
+			id: 'setVar',
+			type: 'textinput',
+			label: label,
+			default: `${def}`,
+			regex: Regex.SOMETHING,
```

Comment:
What should that do?

Reply:
The regex? Ensure it is not an empty field.

Reply:
Given all validation needs to happen in the code after variable parsing, the main or only value is as a UI reminder to not leave the field empty.